### PR TITLE
CPUID.CPU-Z version 2.05

### PR DESCRIPTION
**Pull request by BigThunderSR**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98117)

### CODE REVIEW

File: manifests/c/CPUID/CPU-Z/2.05/CPUID.CPU-Z.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+- processor
+- system
+- system-utility
+- utility
+ReleaseNotes: |-
+  - Intel Core i9-13900/K/F/KF, Core i7-13700/K/F/KF, Core i5-13600/K/F/KF, Core i5-13500, 
+    Core i5-13400 and Core i3-13100
+  - Intel ARC A770, A750, A580 (ACM-G10) and A380 (ACM-G11) GPUs
+  - Support for multipliers > 64x on AMD Ryzen 7000
+  - Preliminary support for NVIDIA RTX 4000 GPUs
```
Comment (mdanish-kh) [inline manifests/c/CPUID/CPU-Z/2.05/CPUID.CPU-Z.locale.en-US.yaml:43]:
```suggestion
ReleaseNotes: |-
  - Intel Xeon Sapphire Rapids platform.
  - Preliminary support for AMD Storm Peak platform.
  - Zhaoxin KX-6000G/4 CPU.
  - Intel Core i9-13900/K/F/KF, Core i7-13700/K/F/KF, Core i5-13600/K/F/KF, Core i5-13500, 
    Core i5-13400 and Core i3-13100
  - Intel ARC A770, A750, A580 (ACM-G10) and A380 (ACM-G11) GPUs
  - Support for multipliers > 64x on AMD Ryzen 7000
  - Preliminary support for NVIDIA RTX 4000 GPUs
```
Full ReleaseNotes

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

File: manifests/c/CPUID/CPU-Z/2.05/CPUID.CPU-Z.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+PackageIdentifier: CPUID.CPU-Z
+PackageVersion: 2.05
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
```
Comment (mdanish-kh) [inline manifests/c/CPUID/CPU-Z/2.05/CPUID.CPU-Z.installer.yaml:12]:
```suggestion
UpgradeBehavior: install
ReleaseDate: 2023-03-01
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
If we're adding ReleaseNotes, we should add them exactly as they are written in the website; https://github.com/microsoft/winget-pkgs/pull/98117#discussion_r1122136475

File: manifests/c/CPUID/CPU-Z/2.05/CPUID.CPU-Z.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+ReleaseDate: 2023-03-01
```
Comment (mdanish-kh) [inline manifests/c/CPUID/CPU-Z/2.05/CPUID.CPU-Z.installer.yaml:13]:
```suggestion
```
My apologies, I did not see the ReleaseDate was already present at the end of file

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Review by mdanish-kh — APPROVED** [review]

**Comment by hez2010** [comment]
The sha256 in the manifest is incorrect. Now we can no longer install CPU-Z due to hash mismatch.

**Pull request merged**